db/hints/sync_point.cc

```diff
@@ -71,7 +71,8 @@ static std::vector<sync_point::shard_rps> decode_one_type_v1(uint16_t shard_coun
 
 sync_point sync_point::decode(sstring_view s) {
     bytes raw = base64_decode(s);
-    seastar::simple_memory_input_stream in{reinterpret_cast<const char*>(raw.data()), raw.size()};
+    sstring_view raw_s(reinterpret_cast<char*>(raw.data()), raw.size());
```

Comment:
Nitpicking over the second commit message, there is a typo: "serielize" -> "serialize".